client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -14,7 +14,7 @@ const trafficPaidStats = [
 	'statsCountryViews',
 ];
 
-const featureFlags = [ 'stats/date-control' ];
+const featureFlags = [ 'stats/date-control', 'download-csv' ];
```

Comment:
Do we need the stats prefix?

Reply:
I took the first one from the feature flag name, Will need to align the names soon. So that we don't hard code it into the code as we're currently doing.